dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -40,41 +41,90 @@ def _create_profile(self, df_series, options=None):
         :rtype: None
         """
 
+        if len(self._profilers) == 0:
+            return
+
         # convert all the values to string
         df_series = df_series.apply(str)
-        
+
+        multiprocess_flag = True
         selected_col_profiles = None
         if options and isinstance(options, StructuredOptions):
             selected_col_profiles = options.enabled_columns
+            multiprocess_flag = options.multiprocess.is_enabled
+
+        pool = mp.Pool(len(self._profilers))
```

Comment:
This creates the pool workers even if the flag is set to false, I think that might not be proper.